runtime/src/accounts_db.rs

```diff
@@ -279,7 +312,7 @@ impl<'a> LoadedAccount<'a> {
         }
     }
 
-    pub fn account(self) -> AccountSharedData {
+    pub fn take_account(self) -> AccountSharedData {
```

Comment:
As this takes ownership, I think using `take_*` is more natural in rust. Otherwise, this just looks like a simple no side-effect getter.